.travis.yml

```diff
@@ -1,18 +1,16 @@
 language: ruby
 rvm:
   - jruby
-  - 2.0.0
-  - 2.1
-  - 2.2
-  - 2.3.0
+  - 2.1.10
+  - 2.2.6
+  - 2.3.3
```

Comment:
From what I can tell, using 2.x should use the latest patch level version of x. Can we try using `2.0`, `2.1`,`2.2`, and `2.3` to see how it handles each? If any of them fail to use the latest version, I'd say it would be fine to use the patch specification, otherwise it's easier to maintain if we keep it to minor only.